components/kafka/pkg/config/config.go

```diff
@@ -34,6 +35,23 @@ type stringSet = map[string]none
 const (
 	KafkaBootstrapServers = "bootstrap.servers"
 	KafkaDebug            = "debug"
+	KafkaLogLevel         = "log_level"
+)
+
+type sysinfoLogLevel int
```

Comment:
Propose we rename everywhere to sysLogLevel (including in function names)

Reply:
done in [7edde1a](https://github.com/SeldonIO/seldon-core/pull/6310/commits/7edde1a5a3f8fcb55a990899378bafe7a7fc64de)